lib/versions.js

```diff
@@ -1,16 +1,25 @@
 const semver = require('semver')
 
-module.exports.getVersionInfo = lastRelease => {
+module.exports.getVersionInfo = (lastRelease, patchVersions = true) => {
   let lastVersion =
     semver.coerce(lastRelease.tag_name) || semver.coerce(lastRelease.name)
 
   if (!lastVersion) {
     return undefined
   }
 
-  return {
+  let versionInfo = {
     incrementedMajor: semver.inc(lastVersion, 'major', true),
     incrementedMinor: semver.inc(lastVersion, 'minor', true),
     incrementedPatch: semver.inc(lastVersion, 'patch', true)
   }
+
+  if (patchVersions === false) {
+    delete versionInfo.incrementedPatch
+    Object.keys(versionInfo).forEach(function(key) {
```

Comment:
Very minor nitpick, but the rest of the codebase uses arrow functions, maybe you can change this to be an arrow function to match the rest of the codebase?

Reply:
I have arrow function locally, sorry for not catching it 😅